webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/domain/reports/StatusReportTest.scala

```diff
@@ -266,13 +266,15 @@ class StatusReportTest extends Specification {
     }
 
     "performance for compute ReportType" should {
+
       val nbSet = 90
       val sizeSet = 100
 
       // force tests to be sequentials
 
       val initData = buildComplianceLevelSets(nbSet,sizeSet)
 
+      val runTest = System.getProperty("run.perf.tests", "false")  == "true"
```

Comment:
can you use `tests.run.perf` for consitance with `tests.clean.tmp` (and we will need to change `test.postgres` to `tests.run.postgres in next minor)